pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/BaseTransformFunctionTest.java

```diff
@@ -65,6 +65,7 @@
   protected static final String STRING_SV_COLUMN = "stringSV";
   protected static final String INT_MV_COLUMN = "intMV";
   protected static final String TIME_COLUMN = "time";
+  protected static final String DOUBLE2_SV_COLUMN = "double2SV";
```

Comment:
Please reuse the DOUBLE_SV_COLUMN, while you can change its value (line 92) if you need value larger than 1 for tests.